provider-directory/src/test/resources/LocationTestResource/mock-provider-response-address-1.json

```diff
@@ -0,0 +1,27 @@
+{
+    "@odata.context" : "https://dws.ppms.va.gov/v1.0/$metadata#ProviderServices",
+    "value" : [
+        {
+            "Name" : "Razzano Jr, Andrew  - Orthopaedic Surgery - Sports Medicine ",
```

Comment:
I'd like to get away from tests that have their data (expected and actual) in separate message files on disk. Tests are generally easier to read if everything is in one place in the file, with builders used for the data.

FYI, when I worked on the story to convert all of the Swagger examples from JSON to builders, I wrote a small Java program to help with the conversion: https://github.com/jblefkove-va/wizardry/blob/master/src/main/java/JsonToBuilder.java

(There is still some manual work in filling in class names and fixing other compiler errors, but it gets you most of the way there.)